gnosis/safe-react
# Fix: avoid loading safe multiple times

## What it solves
Safe info was loaded 3 times on page load.

## How this PR fixes it
Removes the seemingly useless `isSafeLoaded` and makes sure `useScheduledUpdates` doesn't do an initial loading.

## How to test it
- Load a Safe, it should still work the same
- Switch to another Safe on a different network
- Wait for 15 seconds, Safe info should be polled
